### DISCUSSION TITLE
Correct way to upgrade docker - npx prisma

### DISCUSSION THREAD

**Discussion by drwb** [calcom/docker#55] [General]
The [calendso repo](https://github.com/calendso/calendso) Upgrade instructions say:

> In a production environment, run:
> 
> npx prisma migrate deploy

What is the correct way to do this with a docker installation?

`git pull`

`docker-compose up --build -d`

`docker exec calendso-docker_calendso_1 /bin/bash npx prisma migrate deploy`
?

**Comment by PeerRich** [comment]
I believe this has been fixed? Can you try again?

**Reply by krumware** [reply]
The migration is set to happen automatically at startup
